VoodooI2C/VoodooI2C/VoodooI2CDevice/VoodooI2CDeviceNub.cpp

```diff
@@ -148,16 +148,31 @@ IOReturn VoodooI2CDeviceNub::getAlternativeInterrupt(VoodooI2CACPICRSParser* crs
 
     data->release();
```

Comment:
This was in the old code but can we change it to use `OSSafeReleaseNULL(data)`.

Reply:
Done